{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [],
   "source": [
    "HLA_dir = \"../sequences/HLA_I_set_unique.txt\"\n",
    "HLA_list = [] # for netMHCpan\n",
    "HLA_list_stars = [] # for pop coverage analysis\n",
    "\n",
    "with open(HLA_dir, 'r') as input:\n",
    "    hlas = input.readlines()\n",
    "    for hla in hlas:\n",
    "        HLA_list_stars.append(hla.strip('\\n'))\n",
    "        HLA_list.append(\"\".join(hla.strip('\\n').split(\"*\")))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "\n",
    "prot = \"NS1\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['9']\n",
      "['10', '11', '9']\n"
     ]
    }
   ],
   "source": [
    "import glob\n",
    "\n",
    "def fetch_files(protein):\n",
    "    conserved_epitopes = glob.glob(\"../netmhc_binding_inputs/{}/{}_conserved*\".format(prot, prot))\n",
    "    exp_epitopes = glob.glob(\"../netmhc_binding_inputs/{}/{}_exp*\".format(prot, prot))\n",
    "\n",
    "    exp_mers = [i.split(\"_\")[-2] for i in exp_epitopes]\n",
    "    known_mers = [i.split(\"_\")[-2] for i in conserved_epitopes]\n",
    "\n",
    "    return conserved_epitopes, known_mers, exp_epitopes, exp_mers\n",
    "\n",
    "_, kmers, _, emers = fetch_files(prot)\n",
    "\n",
    "print(kmers)\n",
    "print(emers)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [],
   "source": [
    "def run_mhcbinding(prot, file_path, epitopes, status, HLA, mers):\n",
    "    \n",
    "    netMHCpan = \"/mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/netMHCpan\"\n",
    "\n",
    "    for i in range(len(mers)):\n",
    "        hla_name = HLA.replace(\":\", \"_\")\n",
    "        xlsout = \"{}/{}_{}_{}_mers_{}.xls\".format(file_path, prot, status, mers[i], hla_name)\n",
    "        run_netmhcpan = \"{} -f {} -xls -a {} -l {} -xlsfile {}\".format(netMHCpan, epitopes[i], HLA, mers[i], xlsout)\n",
    "\n",
    "        os.system(run_netmhcpan)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-A01:01 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A01_01.xls\n",
      "# Tue Nov 23 13:51:17 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A01:01           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A01_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A01:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A01_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanIZoDvH\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A01:01 : Distance to training data  0.000 (using nearest neighbor HLA-A01:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*01:01     DAPFLDRLRR DAPFLDLRR  0  6  1  0  0   DAPFLDRLRR        cand_0_o 0.0005460   26.459\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-A*01:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-A01:01 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A01_01.xls\n",
      "# Tue Nov 23 13:51:17 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A01:01           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A01_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A01:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A01_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanRX12T5\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A01:01 : Distance to training data  0.000 (using nearest neighbor HLA-A01:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*01:01    LRAFTEEGAIV RAFEEGAIV  1  3  1  0  0   RAFTEEGAIV        cand_3_o 0.0000710   64.286\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-A*01:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*01:01    DAPFLDRLRRD DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_4_o 0.0000160   88.571\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-A*01:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-A01:01 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A01_01.xls\n",
      "# Tue Nov 23 13:51:17 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A01:01           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A01_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A01:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A01_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanOV59I0\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A01:01 : Distance to training data  0.000 (using nearest neighbor HLA-A01:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*01:01      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0006270   24.500\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-A*01:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*01:01      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0008380   20.732\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-A*01:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*01:01      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000020   98.571\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-A*01:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*01:01      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0002420   40.077\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-A*01:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-A02:01 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A02_01.xls\n",
      "# Tue Nov 23 13:51:18 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A02:01           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A02_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A02:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A02_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpaneu7bbK\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A02:01 : Distance to training data  0.000 (using nearest neighbor HLA-A02:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*02:01     DAPFLDRLRR DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_0_o 0.0000200   64.375\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-A*02:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-A02:01 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A02_01.xls\n",
      "# Tue Nov 23 13:51:18 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A02:01           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A02_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A02:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A02_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanA4fEIe\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A02:01 : Distance to training data  0.000 (using nearest neighbor HLA-A02:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*02:01    LRAFTEEGAIV RAFEEGAIV  1  3  1  0  0   RAFTEEGAIV        cand_3_o 0.0001330   37.600\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-A*02:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*02:01    DAPFLDRLRRD DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_4_o 0.0000020   90.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-A*02:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-A02:01 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A02_01.xls\n",
      "# Tue Nov 23 13:51:18 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A02:01           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A02_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A02:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A02_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanGSga35\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A02:01 : Distance to training data  0.000 (using nearest neighbor HLA-A02:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*02:01      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0000190   65.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-A*02:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*02:01      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0000730   45.667\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-A*02:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*02:01      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000000  100.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-A*02:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*02:01      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0014290   15.556\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-A*02:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-A02:03 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A02_03.xls\n",
      "# Tue Nov 23 13:51:19 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A02:03           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A02_03.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A02:03           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A02_03.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanyDSndQ\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A02:03 : Distance to training data  0.000 (using nearest neighbor HLA-A02:03)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*02:03     DAPFLDRLRR DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_0_o 0.0000170   68.571\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-A*02:03. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-A02:03 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A02_03.xls\n",
      "# Tue Nov 23 13:51:19 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A02:03           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A02_03.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A02:03           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A02_03.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanZLOypj\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A02:03 : Distance to training data  0.000 (using nearest neighbor HLA-A02:03)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*02:03    LRAFTEEGAIV RAFEEGAIV  1  3  1  0  0   RAFTEEGAIV        cand_3_o 0.0001840   36.067\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-A*02:03. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*02:03    DAPFLDRLRRD DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_4_o 0.0000020   90.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-A*02:03. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-A02:03 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A02_03.xls\n",
      "# Tue Nov 23 13:51:20 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A02:03           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A02_03.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A02:03           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A02_03.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpannUNHM8\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A02:03 : Distance to training data  0.000 (using nearest neighbor HLA-A02:03)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*02:03      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0000160   69.286\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-A*02:03. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*02:03      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0000330   59.231\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-A*02:03. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*02:03      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000010   95.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-A*02:03. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*02:03      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0024750   13.794\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-A*02:03. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-A02:06 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A02_06.xls\n",
      "# Tue Nov 23 13:51:20 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A02:06           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A02_06.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A02:06           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A02_06.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanSbS4dU\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A02:06 : Distance to training data  0.000 (using nearest neighbor HLA-A02:06)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*02:06     DAPFLDRLRR DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_0_o 0.0000490   63.500\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-A*02:06. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-A02:06 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A02_06.xls\n",
      "# Tue Nov 23 13:51:20 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A02:06           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A02_06.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A02:06           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A02_06.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanzYl7Nk\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A02:06 : Distance to training data  0.000 (using nearest neighbor HLA-A02:06)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*02:06    LRAFTEEGAIV RAFEEGAIV  1  3  1  0  0   RAFTEEGAIV        cand_3_o 0.0004720   33.439\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-A*02:06. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*02:06    DAPFLDRLRRD DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_4_o 0.0000040   90.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-A*02:06. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-A02:06 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A02_06.xls\n",
      "# Tue Nov 23 13:51:21 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A02:06           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A02_06.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A02:06           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A02_06.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanprIqYb\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A02:06 : Distance to training data  0.000 (using nearest neighbor HLA-A02:06)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*02:06      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0000980   53.875\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-A*02:06. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*02:06      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0001620   46.917\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-A*02:06. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*02:06      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000000  100.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-A*02:06. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*02:06      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0029120   16.465\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-A*02:06. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-A03:01 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A03_01.xls\n",
      "# Tue Nov 23 13:51:21 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A03:01           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A03_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A03:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A03_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanNJxlb2\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A03:01 : Distance to training data  0.000 (using nearest neighbor HLA-A03:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*03:01     DAPFLDRLRR DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_0_o 0.0003860   20.217\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-A*03:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-A03:01 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A03_01.xls\n",
      "# Tue Nov 23 13:51:22 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A03:01           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A03_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A03:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A03_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpandNpTDp\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A03:01 : Distance to training data  0.000 (using nearest neighbor HLA-A03:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*03:01    LRAFTEEGAIV RAFEEGAIV  1  3  1  0  0   RAFTEEGAIV        cand_3_o 0.0000060   80.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-A*03:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*03:01    DAPFLDRLRRD DPFLDRLRR  0  1  1  0  0   DAPFLDRLRR        cand_4_o 0.0000180   61.250\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-A*03:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-A03:01 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A03_01.xls\n",
      "# Tue Nov 23 13:51:22 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A03:01           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A03_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A03:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A03_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanETnYQj\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A03:01 : Distance to training data  0.000 (using nearest neighbor HLA-A03:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*03:01      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0001640   28.133\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-A*03:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*03:01      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0041860    7.914\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-A*03:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*03:01      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000020   90.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-A*03:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*03:01      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0000290   53.125\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-A*03:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-A11:01 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A11_01.xls\n",
      "# Tue Nov 23 13:51:23 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A11:01           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A11_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A11:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A11_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanACGge5\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A11:01 : Distance to training data  0.000 (using nearest neighbor HLA-A11:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*11:01     DAPFLDRLRR DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_0_o 0.0005610   13.964\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-A*11:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-A11:01 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A11_01.xls\n",
      "# Tue Nov 23 13:51:23 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A11:01           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A11_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A11:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A11_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanSIcNCw\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A11:01 : Distance to training data  0.000 (using nearest neighbor HLA-A11:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*11:01    LRAFTEEGAIV RAFEEGAIV  1  3  1  0  0   RAFTEEGAIV        cand_3_o 0.0000010   90.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-A*11:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*11:01    DAPFLDRLRRD DPFLDRLRR  0  1  1  0  0   DAPFLDRLRR        cand_4_o 0.0000130   52.500\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-A*11:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-A11:01 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A11_01.xls\n",
      "# Tue Nov 23 13:51:23 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A11:01           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A11_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A11:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A11_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpan1uyC6o\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A11:01 : Distance to training data  0.000 (using nearest neighbor HLA-A11:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*11:01      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0005200   14.353\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-A*11:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*11:01      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0039930    7.199\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-A*11:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*11:01      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000000  100.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-A*11:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*11:01      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0000150   50.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-A*11:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-A23:01 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A23_01.xls\n",
      "# Tue Nov 23 13:51:24 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A23:01           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A23_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A23:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A23_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanxcMJq8\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A23:01 : Distance to training data  0.000 (using nearest neighbor HLA-A23:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*23:01     DAPFLDRLRR DAPFLDRLR  0  8  1  0  0   DAPFLDRLRR        cand_0_o 0.0000080   58.333\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-A*23:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-A23:01 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A23_01.xls\n",
      "# Tue Nov 23 13:51:24 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A23:01           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A23_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A23:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A23_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanpWcliy\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A23:01 : Distance to training data  0.000 (using nearest neighbor HLA-A23:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*23:01    LRAFTEEGAIV LFTEEGAIV  0  1  2  0  0  LRAFTEEGAIV        cand_3_o 0.0000650   31.500\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-A*23:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*23:01    DAPFLDRLRRD DFLDRLRRD  0  1  2  0  0  DAPFLDRLRRD        cand_4_o 0.0000020   80.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-A*23:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-A23:01 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A23_01.xls\n",
      "# Tue Nov 23 13:51:24 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A23:01           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A23_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A23:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A23_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanyqUrIs\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A23:01 : Distance to training data  0.000 (using nearest neighbor HLA-A23:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*23:01      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0000290   41.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-A*23:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*23:01      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0000310   40.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-A*23:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*23:01      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000450   35.500\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-A*23:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*23:01      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0019810    8.504\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-A*23:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-A24:02 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A24_02.xls\n",
      "# Tue Nov 23 13:51:25 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A24:02           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A24_02.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A24:02           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A24_02.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpan4TX81b\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A24:02 : Distance to training data  0.000 (using nearest neighbor HLA-A24:02)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*24:02     DAPFLDRLRR DAPFLRLRR  0  5  1  0  0   DAPFLDRLRR        cand_0_o 0.0000060   60.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-A*24:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-A24:02 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A24_02.xls\n",
      "# Tue Nov 23 13:51:25 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A24:02           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A24_02.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A24:02           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A24_02.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanBeCY7F\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A24:02 : Distance to training data  0.000 (using nearest neighbor HLA-A24:02)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*24:02    LRAFTEEGAIV LFTEEGAIV  0  1  2  0  0  LRAFTEEGAIV        cand_3_o 0.0000610   31.167\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-A*24:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*24:02    DAPFLDRLRRD DFLDRLRRD  0  1  2  0  0  DAPFLDRLRRD        cand_4_o 0.0000020   80.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-A*24:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-A24:02 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A24_02.xls\n",
      "# Tue Nov 23 13:51:25 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A24:02           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A24_02.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A24:02           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A24_02.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanCGwVow\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A24:02 : Distance to training data  0.000 (using nearest neighbor HLA-A24:02)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*24:02      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0000280   40.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-A*24:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*24:02      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0000280   40.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-A*24:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*24:02      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000320   38.333\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-A*24:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*24:02      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0022110    7.942\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-A*24:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-A26:01 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A26_01.xls\n",
      "# Tue Nov 23 13:51:26 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A26:01           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A26_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A26:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A26_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanLzcB6j\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A26:01 : Distance to training data  0.000 (using nearest neighbor HLA-A26:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*26:01     DAPFLDRLRR DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_0_o 0.0036670    7.935\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-A*26:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-A26:01 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A26_01.xls\n",
      "# Tue Nov 23 13:51:26 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A26:01           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A26_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A26:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A26_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpann9x4iK\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A26:01 : Distance to training data  0.000 (using nearest neighbor HLA-A26:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*26:01    LRAFTEEGAIV LRAEEGAIV  0  3  2  0  0  LRAFTEEGAIV        cand_3_o 0.0000190   59.286\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-A*26:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*26:01    DAPFLDRLRRD DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_4_o 0.0000340   50.909\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-A*26:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-A26:01 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A26_01.xls\n",
      "# Tue Nov 23 13:51:27 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A26:01           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A26_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A26:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A26_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpan1FuGLC\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A26:01 : Distance to training data  0.000 (using nearest neighbor HLA-A26:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*26:01      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0058590    6.220\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-A*26:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*26:01      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0011920   13.459\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-A*26:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*26:01      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000010   95.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-A*26:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*26:01      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0001720   29.938\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-A*26:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-A30:01 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A30_01.xls\n",
      "# Tue Nov 23 13:51:27 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A30:01           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A30_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A30:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A30_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanV3wIPm\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A30:01 : Distance to training data  0.000 (using nearest neighbor HLA-A30:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*30:01     DAPFLDRLRR DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_0_o 0.0006180   40.581\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-A*30:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-A30:01 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A30_01.xls\n",
      "# Tue Nov 23 13:51:27 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A30:01           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A30_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A30:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A30_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanukAcnM\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A30:01 : Distance to training data  0.000 (using nearest neighbor HLA-A30:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*30:01    LRAFTEEGAIV RAFEEGAIV  1  3  1  0  0   RAFTEEGAIV        cand_3_o 0.0001440   64.490\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-A*30:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*30:01    DAPFLDRLRRD DPFLDRLRR  0  1  1  0  0   DAPFLDRLRR        cand_4_o 0.0000250   88.214\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-A*30:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-A30:01 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A30_01.xls\n",
      "# Tue Nov 23 13:51:28 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A30:01           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A30_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A30:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A30_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanmTwnjF\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A30:01 : Distance to training data  0.000 (using nearest neighbor HLA-A30:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*30:01      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0007370   37.925\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-A*30:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*30:01      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0124030    9.543\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-A*30:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*30:01      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000430   82.353\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-A*30:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*30:01      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0010180   33.352\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-A*30:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-A30:02 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A30_02.xls\n",
      "# Tue Nov 23 13:51:28 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A30:02           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A30_02.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A30:02           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A30_02.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpancbAD8s\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A30:02 : Distance to training data  0.000 (using nearest neighbor HLA-A30:02)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*30:02     DAPFLDRLRR DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_0_o 0.0001670   59.464\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-A*30:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-A30:02 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A30_02.xls\n",
      "# Tue Nov 23 13:51:29 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A30:02           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A30_02.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A30:02           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A30_02.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanEqCgVS\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A30:02 : Distance to training data  0.000 (using nearest neighbor HLA-A30:02)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*30:02    LRAFTEEGAIV RAFEEGAIV  1  3  1  0  0   RAFTEEGAIV        cand_3_o 0.0001700   59.196\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-A*30:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*30:02    DAPFLDRLRRD DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_4_o 0.0000050   96.875\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-A*30:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-A30:02 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A30_02.xls\n",
      "# Tue Nov 23 13:51:29 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A30:02           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A30_02.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A30:02           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A30_02.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpan4VYIVK\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A30:02 : Distance to training data  0.000 (using nearest neighbor HLA-A30:02)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*30:02      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0001070   66.875\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-A*30:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*30:02      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0013260   26.874\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-A*30:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*30:02      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000100   93.889\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-A*30:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*30:02      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0014510   25.690\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-A*30:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-A31:01 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A31_01.xls\n",
      "# Tue Nov 23 13:51:29 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A31:01           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A31_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A31:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A31_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpan3TEmMx\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A31:01 : Distance to training data  0.000 (using nearest neighbor HLA-A31:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*31:01     DAPFLDRLRR DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_0_o 0.0082970    8.220\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-A*31:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-A31:01 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A31_01.xls\n",
      "# Tue Nov 23 13:51:30 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A31:01           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A31_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A31:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A31_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanVmHccW\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A31:01 : Distance to training data  0.000 (using nearest neighbor HLA-A31:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*31:01    LRAFTEEGAIV RAFEEGAIV  1  3  1  0  0   RAFTEEGAIV        cand_3_o 0.0000100   76.667\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-A*31:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*31:01    DAPFLDRLRRD DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_4_o 0.0001150   40.333\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-A*31:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-A31:01 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A31_01.xls\n",
      "# Tue Nov 23 13:51:30 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A31:01           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A31_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A31:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A31_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpans6VnkO\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A31:01 : Distance to training data  0.000 (using nearest neighbor HLA-A31:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*31:01      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0089010    7.981\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-A*31:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*31:01      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0423760    3.971\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-A*31:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*31:01      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000130   73.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-A*31:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*31:01      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0000710   47.200\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-A*31:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-A32:01 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A32_01.xls\n",
      "# Tue Nov 23 13:51:31 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A32:01           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A32_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A32:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A32_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanuDN2nC\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A32:01 : Distance to training data  0.000 (using nearest neighbor HLA-A32:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*32:01     DAPFLDRLRR DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_0_o 0.0000180   53.750\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-A*32:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-A32:01 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A32_01.xls\n",
      "# Tue Nov 23 13:51:31 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A32:01           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A32_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A32:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A32_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpankRTlE1\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A32:01 : Distance to training data  0.000 (using nearest neighbor HLA-A32:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*32:01    LRAFTEEGAIV RAFEEGAIV  1  3  1  0  0   RAFTEEGAIV        cand_3_o 0.0000160   55.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-A*32:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*32:01    DAPFLDRLRRD DPFLDRLRR  0  1  1  0  0   DAPFLDRLRR        cand_4_o 0.0000010   90.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-A*32:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-A32:01 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A32_01.xls\n",
      "# Tue Nov 23 13:51:31 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A32:01           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A32_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A32:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A32_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanmzhr3U\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A32:01 : Distance to training data  0.000 (using nearest neighbor HLA-A32:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*32:01      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0000160   55.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-A*32:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*32:01      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0002610   24.613\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-A*32:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*32:01      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000000  100.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-A*32:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*32:01      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0000470   42.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-A*32:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-A33:01 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A33_01.xls\n",
      "# Tue Nov 23 13:51:32 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A33:01           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A33_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A33:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A33_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpaneocsyG\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A33:01 : Distance to training data  0.000 (using nearest neighbor HLA-A33:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*33:01     DAPFLDRLRR DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_0_o 0.3088610    0.522 <= WB\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-A*33:01. Number of high binders 0. Number of weak binders 1. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-A33:01 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A33_01.xls\n",
      "# Tue Nov 23 13:51:32 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A33:01           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A33_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A33:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A33_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanu4Wvp5\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A33:01 : Distance to training data  0.000 (using nearest neighbor HLA-A33:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*33:01    LRAFTEEGAIV LFTEEGAIV  0  1  2  0  0  LRAFTEEGAIV        cand_3_o 0.0000220   68.571\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-A*33:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*33:01    DAPFLDRLRRD DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_4_o 0.0051080    7.920\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-A*33:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-A33:01 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A33_01.xls\n",
      "# Tue Nov 23 13:51:32 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A33:01           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A33_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A33:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A33_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanNvFOoV\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A33:01 : Distance to training data  0.000 (using nearest neighbor HLA-A33:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*33:01      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.2801150    0.605 <= WB\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-A*33:01. Number of high binders 0. Number of weak binders 1. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*33:01      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.1247410    1.393 <= WB\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-A*33:01. Number of high binders 0. Number of weak binders 1. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*33:01      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000050   88.333\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-A*33:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*33:01      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0000750   47.500\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-A*33:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-A68:01 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A68_01.xls\n",
      "# Tue Nov 23 13:51:33 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A68:01           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A68_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A68:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A68_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXjCLtI\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A68:01 : Distance to training data  0.000 (using nearest neighbor HLA-A68:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*68:01     DAPFLDRLRR DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_0_o 0.4763470    0.839 <= WB\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-A*68:01. Number of high binders 0. Number of weak binders 1. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-A68:01 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A68_01.xls\n",
      "# Tue Nov 23 13:51:33 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A68:01           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A68_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A68:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A68_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanw3ZN8b\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A68:01 : Distance to training data  0.000 (using nearest neighbor HLA-A68:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*68:01    LRAFTEEGAIV RAFEEGAIV  1  3  1  0  0   RAFTEEGAIV        cand_3_o 0.0000040   82.500\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-A*68:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*68:01    DAPFLDRLRRD DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_4_o 0.0051730   10.413\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-A*68:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-A68:01 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A68_01.xls\n",
      "# Tue Nov 23 13:51:33 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A68:01           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A68_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A68:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A68_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanZzkyS0\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A68:01 : Distance to training data  0.000 (using nearest neighbor HLA-A68:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*68:01      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.4102010    1.003 <= WB\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-A*68:01. Number of high binders 0. Number of weak binders 1. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*68:01      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.1802450    2.086\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-A*68:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*68:01      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000000  100.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-A*68:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*68:01      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0000140   64.167\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-A*68:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-A68:02 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A68_02.xls\n",
      "# Tue Nov 23 13:51:34 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A68:02           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A68_02.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A68:02           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-A68_02.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanvIhdKM\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A68:02 : Distance to training data  0.000 (using nearest neighbor HLA-A68:02)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*68:02     DAPFLDRLRR DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_0_o 0.0009290   20.583\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-A*68:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-A68:02 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A68_02.xls\n",
      "# Tue Nov 23 13:51:34 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A68:02           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A68_02.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A68:02           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-A68_02.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanQGJkOc\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A68:02 : Distance to training data  0.000 (using nearest neighbor HLA-A68:02)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*68:02    LRAFTEEGAIV RAFEEGAIV  1  3  1  0  0   RAFTEEGAIV        cand_3_o 0.0001570   38.857\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-A*68:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*68:02    DAPFLDRLRRD DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_4_o 0.0000190   66.429\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-A*68:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-A68:02 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A68_02.xls\n",
      "# Tue Nov 23 13:51:35 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-A68:02           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A68_02.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-A68:02           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-A68_02.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpancgqI56\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-A68:02 : Distance to training data  0.000 (using nearest neighbor HLA-A68:02)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*68:02      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0025410   13.444\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-A*68:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*68:02      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0008480   21.344\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-A*68:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*68:02      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000000  100.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-A*68:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-A*68:02      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0009020   20.817\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-A*68:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-B07:02 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B07_02.xls\n",
      "# Tue Nov 23 13:51:35 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B07:02           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B07_02.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B07:02           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B07_02.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanpsOFmR\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B07:02 : Distance to training data  0.000 (using nearest neighbor HLA-B07:02)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*07:02     DAPFLDRLRR APFLDRLRR  1  0  0  0  0    APFLDRLRR        cand_0_o 0.0003530   23.795\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-B*07:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-B07:02 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B07_02.xls\n",
      "# Tue Nov 23 13:51:35 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B07:02           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B07_02.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B07:02           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B07_02.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpan1G1Vlk\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B07:02 : Distance to training data  0.000 (using nearest neighbor HLA-B07:02)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*07:02    LRAFTEEGAIV LRAEEGAIV  0  3  2  0  0  LRAFTEEGAIV        cand_3_o 0.0000680   44.750\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-B*07:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*07:02    DAPFLDRLRRD DPFLDRLRR  0  1  1  0  0   DAPFLDRLRR        cand_4_o 0.0000080   78.333\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-B*07:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-B07:02 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B07_02.xls\n",
      "# Tue Nov 23 13:51:36 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B07:02           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B07_02.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B07:02           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B07_02.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanQHMmgb\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B07:02 : Distance to training data  0.000 (using nearest neighbor HLA-B07:02)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*07:02      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0000450   51.154\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-B*07:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*07:02      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0234590    3.306\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-B*07:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*07:02      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000000  100.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-B*07:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*07:02      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0003970   22.690\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-B*07:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-B08:01 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B08_01.xls\n",
      "# Tue Nov 23 13:51:36 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B08:01           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B08_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B08:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B08_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanJDp4bW\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B08:01 : Distance to training data  0.000 (using nearest neighbor HLA-B08:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*08:01     DAPFLDRLRR DPFLDRLRR  0  1  1  0  0   DAPFLDRLRR        cand_0_o 0.0001640   49.100\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-B*08:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-B08:01 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B08_01.xls\n",
      "# Tue Nov 23 13:51:36 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B08:01           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B08_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B08:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B08_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanS8wBTo\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B08:01 : Distance to training data  0.000 (using nearest neighbor HLA-B08:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*08:01    LRAFTEEGAIV LFTEEGAIV  0  1  2  0  0  LRAFTEEGAIV        cand_3_o 0.0001210   54.146\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-B*08:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*08:01    DAPFLDRLRRD DPFLDRLRR  0  1  1  0  0   DAPFLDRLRR        cand_4_o 0.0000250   79.500\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-B*08:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-B08:01 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B08_01.xls\n",
      "# Tue Nov 23 13:51:37 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B08:01           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B08_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B08:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B08_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanhdmQUd\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B08:01 : Distance to training data  0.000 (using nearest neighbor HLA-B08:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*08:01      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0002800   40.750\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-B*08:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*08:01      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0015700   20.328\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-B*08:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*08:01      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000120   88.571\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-B*08:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*08:01      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0012310   22.592\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-B*08:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-B15:01 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B15_01.xls\n",
      "# Tue Nov 23 13:51:37 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B15:01           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B15_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B15:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B15_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanhWJJ81\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B15:01 : Distance to training data  0.000 (using nearest neighbor HLA-B15:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*15:01     DAPFLDRLRR DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_0_o 0.0000240   63.125\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-B*15:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-B15:01 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B15_01.xls\n",
      "# Tue Nov 23 13:51:38 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B15:01           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B15_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B15:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B15_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanWi886p\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B15:01 : Distance to training data  0.000 (using nearest neighbor HLA-B15:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*15:01    LRAFTEEGAIV LRAEEGAIV  0  3  2  0  0  LRAFTEEGAIV        cand_3_o 0.0000290   60.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-B*15:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*15:01    DAPFLDRLRRD DPFLDRLRR  0  1  1  0  0   DAPFLDRLRR        cand_4_o 0.0000020   90.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-B*15:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-B15:01 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B15_01.xls\n",
      "# Tue Nov 23 13:51:38 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B15:01           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B15_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B15:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B15_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanQ6AMVj\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B15:01 : Distance to training data  0.000 (using nearest neighbor HLA-B15:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*15:01      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0000180   67.500\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-B*15:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*15:01      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0001200   40.111\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-B*15:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*15:01      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000000  100.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-B*15:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*15:01      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0002490   31.143\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-B*15:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-B35:01 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B35_01.xls\n",
      "# Tue Nov 23 13:51:38 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B35:01           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B35_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B35:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B35_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanHH5zJ5\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B35:01 : Distance to training data  0.000 (using nearest neighbor HLA-B35:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*35:01     DAPFLDRLRR DPFLDRLRR  0  1  1  0  0   DAPFLDRLRR        cand_0_o 0.0015210   11.766\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-B*35:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-B35:01 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B35_01.xls\n",
      "# Tue Nov 23 13:51:39 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B35:01           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B35_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B35:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B35_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanOTZYLu\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B35:01 : Distance to training data  0.000 (using nearest neighbor HLA-B35:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*35:01    LRAFTEEGAIV LFTEEGAIV  0  1  2  0  0  LRAFTEEGAIV        cand_3_o 0.0000070   67.500\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-B*35:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*35:01    DAPFLDRLRRD DPFLDRLRR  0  1  1  0  0   DAPFLDRLRR        cand_4_o 0.0000340   44.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-B*35:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-B35:01 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B35_01.xls\n",
      "# Tue Nov 23 13:51:39 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B35:01           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B35_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B35:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B35_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpan7KrkAn\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B35:01 : Distance to training data  0.000 (using nearest neighbor HLA-B35:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*35:01      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0008060   15.039\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-B*35:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*35:01      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0069240    6.279\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-B*35:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*35:01      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000010   90.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-B*35:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*35:01      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0001030   31.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-B*35:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-B38:01 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B38_01.xls\n",
      "# Tue Nov 23 13:51:40 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B38:01           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B38_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B38:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B38_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanc0EKh9\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B38:01 : Distance to training data  0.000 (using nearest neighbor HLA-B38:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*38:01     DAPFLDRLRR DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_0_o 0.0000230   56.875\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-B*38:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-B38:01 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B38_01.xls\n",
      "# Tue Nov 23 13:51:40 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B38:01           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B38_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B38:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B38_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanmTLVaz\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B38:01 : Distance to training data  0.000 (using nearest neighbor HLA-B38:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*38:01    LRAFTEEGAIV LRAEEGAIV  0  3  2  0  0  LRAFTEEGAIV        cand_3_o 0.0005880   20.427\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-B*38:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*38:01    DAPFLDRLRRD DALDRLRRD  0  2  2  0  0  DAPFLDRLRRD        cand_4_o 0.0000010   95.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-B*38:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-B38:01 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B38_01.xls\n",
      "# Tue Nov 23 13:51:40 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B38:01           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B38_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B38:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B38_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanZ6Gkuu\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B38:01 : Distance to training data  0.000 (using nearest neighbor HLA-B38:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*38:01      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0001430   33.417\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-B*38:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*38:01      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0002540   27.704\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-B*38:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*38:01      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000010   95.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-B*38:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*38:01      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0033340    9.789\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-B*38:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-B40:01 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B40_01.xls\n",
      "# Tue Nov 23 13:51:41 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B40:01           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B40_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B40:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B40_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanpdXTig\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B40:01 : Distance to training data  0.000 (using nearest neighbor HLA-B40:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*40:01     DAPFLDRLRR APFLDRLRR  1  0  0  0  0    APFLDRLRR        cand_0_o 0.0000070   57.500\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-B*40:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-B40:01 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B40_01.xls\n",
      "# Tue Nov 23 13:51:41 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B40:01           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B40_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B40:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B40_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpann1hvXD\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B40:01 : Distance to training data  0.000 (using nearest neighbor HLA-B40:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*40:01    LRAFTEEGAIV LRAEEGAIV  0  3  2  0  0  LRAFTEEGAIV        cand_3_o 0.0000730   27.714\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-B*40:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*40:01    DAPFLDRLRRD DPFLDRLRR  0  1  1  0  0   DAPFLDRLRR        cand_4_o 0.0000000  100.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-B*40:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-B40:01 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B40_01.xls\n",
      "# Tue Nov 23 13:51:41 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B40:01           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B40_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B40:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B40_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanqKGA1v\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B40:01 : Distance to training data  0.000 (using nearest neighbor HLA-B40:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*40:01      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0000200   43.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-B*40:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*40:01      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0001220   22.733\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-B*40:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*40:01      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000000  100.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-B*40:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*40:01      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0004170   13.658\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-B*40:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-B44:02 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B44_02.xls\n",
      "# Tue Nov 23 13:51:42 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B44:02           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B44_02.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B44:02           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B44_02.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpankrEVVi\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B44:02 : Distance to training data  0.000 (using nearest neighbor HLA-B44:02)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*44:02     DAPFLDRLRR DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_0_o 0.0000410   40.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-B*44:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-B44:02 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B44_02.xls\n",
      "# Tue Nov 23 13:51:42 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B44:02           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B44_02.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B44:02           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B44_02.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanIAsAgL\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B44:02 : Distance to training data  0.000 (using nearest neighbor HLA-B44:02)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*44:02    LRAFTEEGAIV LRAEEGAIV  0  3  2  0  0  LRAFTEEGAIV        cand_3_o 0.0000130   60.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-B*44:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*44:02    DAPFLDRLRRD APLDRLRRD  1  2  1  0  0   APFLDRLRRD        cand_4_o 0.0000050   75.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-B*44:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-B44:02 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B44_02.xls\n",
      "# Tue Nov 23 13:51:42 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B44:02           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B44_02.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B44:02           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B44_02.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanD5j5AE\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B44:02 : Distance to training data  0.000 (using nearest neighbor HLA-B44:02)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*44:02      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0000540   36.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-B*44:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*44:02      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0010910    9.971\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-B*44:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*44:02      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000000  100.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-B*44:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*44:02      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0002930   17.825\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-B*44:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-B44:03 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B44_03.xls\n",
      "# Tue Nov 23 13:51:43 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B44:03           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B44_03.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B44:03           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B44_03.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanr1cuqn\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B44:03 : Distance to training data  0.000 (using nearest neighbor HLA-B44:03)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*44:03     DAPFLDRLRR DPFLDRLRR  0  1  1  0  0   DAPFLDRLRR        cand_0_o 0.0000460   36.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-B*44:03. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-B44:03 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B44_03.xls\n",
      "# Tue Nov 23 13:51:43 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B44:03           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B44_03.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B44:03           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B44_03.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpansqHRxR\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B44:03 : Distance to training data  0.000 (using nearest neighbor HLA-B44:03)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*44:03    LRAFTEEGAIV LRAEEGAIV  0  3  2  0  0  LRAFTEEGAIV        cand_3_o 0.0000120   57.500\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-B*44:03. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*44:03    DAPFLDRLRRD APFDRLRRD  1  3  1  0  0   APFLDRLRRD        cand_4_o 0.0000030   80.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-B*44:03. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-B44:03 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B44_03.xls\n",
      "# Tue Nov 23 13:51:44 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B44:03           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B44_03.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B44:03           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B44_03.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanvrfzkG\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B44:03 : Distance to training data  0.000 (using nearest neighbor HLA-B44:03)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*44:03      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0000870   28.143\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-B*44:03. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*44:03      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0020910    7.621\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-B*44:03. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*44:03      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000000  100.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-B*44:03. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*44:03      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0002150   19.630\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-B*44:03. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-B51:01 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B51_01.xls\n",
      "# Tue Nov 23 13:51:44 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B51:01           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B51_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B51:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B51_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanu48zAr\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B51:01 : Distance to training data  0.000 (using nearest neighbor HLA-B51:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*51:01     DAPFLDRLRR DPFLDRLRR  0  1  1  0  0   DAPFLDRLRR        cand_0_o 0.0010830   23.991\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-B*51:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-B51:01 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B51_01.xls\n",
      "# Tue Nov 23 13:51:44 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B51:01           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B51_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B51:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B51_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanAedcRU\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B51:01 : Distance to training data  0.000 (using nearest neighbor HLA-B51:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*51:01    LRAFTEEGAIV RAFEEGAIV  1  3  1  0  0   RAFTEEGAIV        cand_3_o 0.0007440   27.724\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-B*51:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*51:01    DAPFLDRLRRD DPFLDRLRR  0  1  1  0  0   DAPFLDRLRR        cand_4_o 0.0000680   58.400\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-B*51:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-B51:01 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B51_01.xls\n",
      "# Tue Nov 23 13:51:45 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B51:01           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B51_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B51:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B51_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpan3jdBJM\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B51:01 : Distance to training data  0.000 (using nearest neighbor HLA-B51:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*51:01      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0019980   18.567\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-B*51:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*51:01      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0020320   18.429\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-B*51:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*51:01      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000010   98.333\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-B*51:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*51:01      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0011920   23.043\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-B*51:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-B53:01 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B53_01.xls\n",
      "# Tue Nov 23 13:51:45 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B53:01           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B53_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B53:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B53_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanyRivRx\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B53:01 : Distance to training data  0.000 (using nearest neighbor HLA-B53:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*53:01     DAPFLDRLRR DPFLDRLRR  0  1  1  0  0   DAPFLDRLRR        cand_0_o 0.0007480   14.376\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-B*53:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-B53:01 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B53_01.xls\n",
      "# Tue Nov 23 13:51:45 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B53:01           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B53_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B53:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B53_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanKd7lfW\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B53:01 : Distance to training data  0.000 (using nearest neighbor HLA-B53:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*53:01    LRAFTEEGAIV RAFEEGAIV  1  3  1  0  0   RAFTEEGAIV        cand_3_o 0.0000200   50.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-B*53:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*53:01    DAPFLDRLRRD DPFLDRLRR  0  1  1  0  0   DAPFLDRLRR        cand_4_o 0.0000130   57.500\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-B*53:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-B53:01 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B53_01.xls\n",
      "# Tue Nov 23 13:51:46 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B53:01           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B53_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B53:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B53_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanUtRajR\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B53:01 : Distance to training data  0.000 (using nearest neighbor HLA-B53:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*53:01      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0006100   15.550\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-B*53:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*53:01      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0038870    7.240\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-B*53:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*53:01      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000000  100.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-B*53:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*53:01      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0002930   20.462\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-B*53:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-B57:01 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B57_01.xls\n",
      "# Tue Nov 23 13:51:46 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B57:01           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B57_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B57:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B57_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanZhXc4A\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B57:01 : Distance to training data  0.000 (using nearest neighbor HLA-B57:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*57:01     DAPFLDRLRR DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_0_o 0.0002220   44.867\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-B*57:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-B57:01 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B57_01.xls\n",
      "# Tue Nov 23 13:51:47 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B57:01           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B57_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B57:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B57_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpan4SwAw2\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B57:01 : Distance to training data  0.000 (using nearest neighbor HLA-B57:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*57:01    LRAFTEEGAIV RAFEEGAIV  1  3  1  0  0   RAFTEEGAIV        cand_3_o 0.0003340   38.750\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-B*57:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*57:01    DAPFLDRLRRD DALDRLRRD  0  2  2  0  0  DAPFLDRLRRD        cand_4_o 0.0000160   85.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-B*57:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-B57:01 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B57_01.xls\n",
      "# Tue Nov 23 13:51:47 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B57:01           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B57_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B57:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B57_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanuVjjCW\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B57:01 : Distance to training data  0.000 (using nearest neighbor HLA-B57:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*57:01      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0000900   59.167\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-B*57:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*57:01      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0008670   26.651\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-B*57:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*57:01      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000030   97.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-B*57:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*57:01      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0001070   56.333\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-B*57:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-B58:01 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B58_01.xls\n",
      "# Tue Nov 23 13:51:47 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B58:01           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B58_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B58:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-B58_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpan7GGi3S\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B58:01 : Distance to training data  0.000 (using nearest neighbor HLA-B58:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*58:01     DAPFLDRLRR DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_0_o 0.0001360   41.889\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-B*58:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-B58:01 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B58_01.xls\n",
      "# Tue Nov 23 13:51:48 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B58:01           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B58_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B58:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-B58_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanimASX6\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B58:01 : Distance to training data  0.000 (using nearest neighbor HLA-B58:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*58:01    LRAFTEEGAIV RAFEEGAIV  1  3  1  0  0   RAFTEEGAIV        cand_3_o 0.0002440   33.556\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-B*58:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*58:01    DAPFLDRLRRD DPFLDRLRR  0  1  1  0  0   DAPFLDRLRR        cand_4_o 0.0000080   85.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-B*58:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-B58:01 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B58_01.xls\n",
      "# Tue Nov 23 13:51:48 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-B58:01           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B58_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-B58:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-B58_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanHtiiFY\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-B58:01 : Distance to training data  0.000 (using nearest neighbor HLA-B58:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*58:01      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0000700   52.381\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-B*58:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*58:01      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0006070   22.921\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-B*58:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*58:01      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000010   98.333\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-B*58:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-B*58:01      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0000740   51.429\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-B*58:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-C04:01 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-C04_01.xls\n",
      "# Tue Nov 23 13:51:49 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-C04:01           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-C04_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-C04:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-C04_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanR7cnhL\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-C04:01 : Distance to training data  0.000 (using nearest neighbor HLA-C04:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*04:01     DAPFLDRLRR DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_0_o 0.0000220   60.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-C*04:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-C04:01 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-C04_01.xls\n",
      "# Tue Nov 23 13:51:49 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-C04:01           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-C04_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-C04:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-C04_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpan0YxLca\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-C04:01 : Distance to training data  0.000 (using nearest neighbor HLA-C04:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*04:01    LRAFTEEGAIV LFTEEGAIV  0  1  2  0  0  LRAFTEEGAIV        cand_3_o 0.0001100   36.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-C*04:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*04:01    DAPFLDRLRRD DAPLDRLRR  0  3  1  0  0   DAPFLDRLRR        cand_4_o 0.0000010   95.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-C*04:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-C04:01 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-C04_01.xls\n",
      "# Tue Nov 23 13:51:50 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-C04:01           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-C04_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-C04:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-C04_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanl408t2\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-C04:01 : Distance to training data  0.000 (using nearest neighbor HLA-C04:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*04:01      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0000500   47.333\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-C*04:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*04:01      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0003050   24.034\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-C*04:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*04:01      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000040   85.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-C*04:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*04:01      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0162080    2.593\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-C*04:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-C05:01 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-C05_01.xls\n",
      "# Tue Nov 23 13:51:50 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-C05:01           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-C05_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-C05:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-C05_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanxxxnaO\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-C05:01 : Distance to training data  0.000 (using nearest neighbor HLA-C05:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*05:01     DAPFLDRLRR DAPFLRLRR  0  5  1  0  0   DAPFLDRLRR        cand_0_o 0.0000040   70.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-C*05:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-C05:01 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-C05_01.xls\n",
      "# Tue Nov 23 13:51:50 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-C05:01           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-C05_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-C05:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-C05_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpana3T3kd\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-C05:01 : Distance to training data  0.000 (using nearest neighbor HLA-C05:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*05:01    LRAFTEEGAIV RAFEEGAIV  1  3  1  0  0   RAFTEEGAIV        cand_3_o 0.0000470   37.250\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-C*05:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*05:01    DAPFLDRLRRD DAPDRLRRD  0  3  2  0  0  DAPFLDRLRRD        cand_4_o 0.0000000  100.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-C*05:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-C05:01 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-C05_01.xls\n",
      "# Tue Nov 23 13:51:51 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-C05:01           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-C05_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-C05:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-C05_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpan13MJ3a\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-C05:01 : Distance to training data  0.000 (using nearest neighbor HLA-C05:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*05:01      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0000140   53.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-C*05:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*05:01      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0000310   42.333\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-C*05:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*05:01      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000000  100.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-C*05:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*05:01      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0012650   10.846\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-C*05:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-C07:02 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-C07_02.xls\n",
      "# Tue Nov 23 13:51:51 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-C07:02           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-C07_02.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-C07:02           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-C07_02.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanAo12nH\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-C07:02 : Distance to training data  0.000 (using nearest neighbor HLA-C07:02)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*07:02     DAPFLDRLRR DAPFLRLRR  0  5  1  0  0   DAPFLDRLRR        cand_0_o 0.0000160   45.500\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-C*07:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-C07:02 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-C07_02.xls\n",
      "# Tue Nov 23 13:51:51 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-C07:02           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-C07_02.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-C07:02           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-C07_02.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpan98eWd2\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-C07:02 : Distance to training data  0.000 (using nearest neighbor HLA-C07:02)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*07:02    LRAFTEEGAIV LRAEEGAIV  0  3  2  0  0  LRAFTEEGAIV        cand_3_o 0.0000470   32.800\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-C*07:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*07:02    DAPFLDRLRRD DAPLDRLRR  0  3  1  0  0   DAPFLDRLRR        cand_4_o 0.0000010   85.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-C*07:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-C07:02 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-C07_02.xls\n",
      "# Tue Nov 23 13:51:52 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-C07:02           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-C07_02.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-C07:02           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-C07_02.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanBcERVU\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-C07:02 : Distance to training data  0.000 (using nearest neighbor HLA-C07:02)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*07:02      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0000410   34.333\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-C*07:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*07:02      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0001160   24.083\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-C*07:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*07:02      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000040   65.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-C*07:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*07:02      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0008230   10.713\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-C*07:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-C12:03 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-C12_03.xls\n",
      "# Tue Nov 23 13:51:52 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-C12:03           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-C12_03.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-C12:03           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-C12_03.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanipjxkJ\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-C12:03 : Distance to training data  0.000 (using nearest neighbor HLA-C12:03)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*12:03     DAPFLDRLRR DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_0_o 0.0000240   42.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-C*12:03. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-C12:03 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-C12_03.xls\n",
      "# Tue Nov 23 13:51:53 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-C12:03           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-C12_03.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-C12:03           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-C12_03.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanSnuTg7\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-C12:03 : Distance to training data  0.000 (using nearest neighbor HLA-C12:03)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*12:03    LRAFTEEGAIV RAFEEGAIV  1  3  1  0  0   RAFTEEGAIV        cand_3_o 0.0000260   41.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-C*12:03. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*12:03    DAPFLDRLRRD DPFLDRLRR  0  1  1  0  0   DAPFLDRLRR        cand_4_o 0.0000010   85.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-C*12:03. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-C12:03 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-C12_03.xls\n",
      "# Tue Nov 23 13:51:53 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-C12:03           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-C12_03.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-C12:03           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-C12_03.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanAG6CP0\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-C12:03 : Distance to training data  0.000 (using nearest neighbor HLA-C12:03)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*12:03      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0001190   26.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-C*12:03. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*12:03      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0001950   22.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-C*12:03. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*12:03      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000000  100.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-C*12:03. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*12:03      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0003140   18.543\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-C*12:03. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-C15:02 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-C15_02.xls\n",
      "# Tue Nov 23 13:51:53 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-C15:02           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-C15_02.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-C15:02           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-C15_02.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpan6kiZxL\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-C15:02 : Distance to training data  0.000 (using nearest neighbor HLA-C15:02)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*15:02     DAPFLDRLRR DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_0_o 0.0000050   68.333\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-C*15:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-C15:02 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-C15_02.xls\n",
      "# Tue Nov 23 13:51:54 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-C15:02           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-C15_02.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-C15:02           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-C15_02.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanKzyMUa\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-C15:02 : Distance to training data  0.000 (using nearest neighbor HLA-C15:02)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*15:02    LRAFTEEGAIV RAFEEGAIV  1  3  1  0  0   RAFTEEGAIV        cand_3_o 0.0002470   26.185\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-C*15:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*15:02    DAPFLDRLRRD DALDRLRRD  0  2  2  0  0  DAPFLDRLRRD        cand_4_o 0.0000000  100.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-C*15:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-C15:02 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-C15_02.xls\n",
      "# Tue Nov 23 13:51:54 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-C15:02           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-C15_02.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-C15:02           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-C15_02.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanoukZG2\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-C15:02 : Distance to training data  0.000 (using nearest neighbor HLA-C15:02)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*15:02      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0000120   58.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-C*15:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*15:02      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0001110   33.500\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-C*15:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*15:02      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000000  100.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-C*15:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*15:02      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0002930   24.784\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-C*15:02. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa -xls -a HLA-C16:01 -l 10 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-C16_01.xls\n",
      "# Tue Nov 23 13:51:55 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-C16:01           MHC allele\n",
      "# -l       10                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-C16_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-C16:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_10_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          10                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_10_mers_HLA-C16_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpan1P4nX5\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 10\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-C16:01 : Distance to training data  0.000 (using nearest neighbor HLA-C16:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*16:01     DAPFLDRLRR DAFLDRLRR  0  2  1  0  0   DAPFLDRLRR        cand_0_o 0.0000150   48.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_0_o. Allele HLA-C*16:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa -xls -a HLA-C16:01 -l 11 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-C16_01.xls\n",
      "# Tue Nov 23 13:51:55 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-C16:01           MHC allele\n",
      "# -l       11                   Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-C16_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-C16:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_11_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          11                   Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_11_mers_HLA-C16_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanvprIIs\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 11\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-C16:01 : Distance to training data  0.000 (using nearest neighbor HLA-C16:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*16:01    LRAFTEEGAIV RAFEEGAIV  1  3  1  0  0   RAFTEEGAIV        cand_3_o 0.0000130   50.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_3_o. Allele HLA-C*16:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*16:01    DAPFLDRLRRD DPFLDRLRR  0  1  1  0  0   DAPFLDRLRR        cand_4_o 0.0000000  100.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_4_o. Allele HLA-C*16:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/bin/netMHCpan -f ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa -xls -a HLA-C16:01 -l 9 -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-C16_01.xls\n",
      "# Tue Nov 23 13:51:55 2021\n",
      "# User: juhlee\n",
      "# PWD : /mnt/c/Users/jhlee/Desktop/Thesis/works/src\n",
      "# Host: Linux JuHyung 4.4.0-22000-Microsoft x86_64\n",
      "# -f       ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "# -xls     1                    Save output to xls file\n",
      "# -a       HLA-C16:01           MHC allele\n",
      "# -l       9                    Peptide length [8-11] (multiple length with ,)\n",
      "# -xlsfile ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-C16_01.xls Filename for xls dump\n",
      "# Command line parameters set to:\n",
      "#\t[-rdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64 Home directory for NetMHpan\n",
      "#\t[-syn filename]      /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/synlist.bin Synaps file\n",
      "#\t[-v]                 0                    Verbose mode\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\n",
      "#\t[-tdir filename]     /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpanXXXXXX Temporary directory (made with mkdtemp)\n",
      "#\t[-hlapseudo filename] /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\n",
      "#\t[-a line]            HLA-C16:01           MHC allele\n",
      "#\t[-f filename]        ../netmhc_binding_inputs/NS1/NS1_exp_9_mers.fa File name with input\n",
      "#\t[-w]                 0                    w option for webface\n",
      "#\t[-s]                 0                    Sort output on descending affinity\n",
      "#\t[-p]                 0                    Use peptide input\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\n",
      "#\t[-l string]          9                    Peptide length [8-11] (multiple length with ,)\n",
      "#\t[-xls]               1                    Save output to xls file\n",
      "#\t[-xlsfile filename]  ../netmhcpan_binding_result/NS1/NS1_experimental_9_mers_HLA-C16_01.xls Filename for xls dump\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\n",
      "#\t[-thrfmt filename]   /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\n",
      "#\t[-version filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/version File with version information\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\n",
      "#\t[-allname filename]  /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/Linux_x86_64/data/allelenames File with print names for alleles\n",
      "#\t[-BA]                0                    Include Binding affinity prediction\n",
      "\n",
      "# NetMHCpan version 4.1b\n",
      "\n",
      "# Tmpdir made /mnt/c/Users/jhlee/Desktop/Thesis/works/models/netMHCpan-4.1/tmp/netMHCpan9I7RiZ\n",
      "# Input is in FSA format\n",
      "\n",
      "# Peptide length 9\n",
      "\n",
      "# Make EL predictions\n",
      "\n",
      "HLA-C16:01 : Distance to training data  0.000 (using nearest neighbor HLA-C16:01)\n",
      "\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*16:01      DAPFLDRLR DAPFLDRLR  0  0  0  0  0    DAPFLDRLR        cand_1_o 0.0001090   28.692\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_1_o. Allele HLA-C*16:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*16:01      APFLDRLRR APFLDRLRR  0  0  0  0  0    APFLDRLRR        cand_2_o 0.0003210   20.609\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_2_o. Allele HLA-C*16:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*16:01      PFLDRLRRD PFLDRLRRD  0  0  0  0  0    PFLDRLRRD        cand_5_o 0.0000000  100.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_5_o. Allele HLA-C*16:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n",
      "# Rank Threshold for Strong binding peptides   0.500\n",
      "# Rank Threshold for Weak binding peptides   2.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL BindLevel\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "   1 HLA-C*16:01      AFTEEGAIV AFTEEGAIV  0  0  0  0  0    AFTEEGAIV        cand_6_o 0.0002270   23.000\n",
      "---------------------------------------------------------------------------------------------------------------------------\n",
      "\n",
      "Protein cand_6_o. Allele HLA-C*16:01. Number of high binders 0. Number of weak binders 0. Number of peptides 1\n",
      "\n",
      "-----------------------------------------------------------------------------------\n"
     ]
    }
   ],
   "source": [
    "file_path = \"../netmhcpan_binding_result/{}\".format(prot)\n",
    "\n",
    "if not os.path.exists(file_path):\n",
    "  os.mkdir(file_path)\n",
    "\n",
    "_, _, exp_epitopes, exp_mers = fetch_files(prot)\n",
    "\n",
    "for hla in HLA_list:\n",
    "  run_mhcbinding(prot, file_path, exp_epitopes, 'experimental', hla, exp_mers)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "file_path = \"../netmhcpan_tool_final_result/{}\".format(prot)\n",
    "\n",
    "if not os.path.exists(file_path):\n",
    "  os.mkdir(file_path)\n",
    "\n",
    "for hla in HLA_list:\n",
    "  run_mhcbinding(prot, file_path, conserved_epitopes, 'conserved', hla, known_mers)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['HLA-B58:01',\n",
       " 'HLA-C04:01',\n",
       " 'HLA-C05:01',\n",
       " 'HLA-C07:02',\n",
       " 'HLA-C12:03',\n",
       " 'HLA-C15:02',\n",
       " 'HLA-C16:01']"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "HLA_list[-7:]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [],
   "source": [
    "import io\n",
    "from xlwt import Workbook\n",
    "from __future__ import unicode_literals\n",
    "\n",
    "def recover_xls(filename):\n",
    "    file1 = io.open(filename, \"r\")\n",
    "\n",
    "    data = file1.readlines()\n",
    "    data\n",
    "\n",
    "    xldoc = Workbook()\n",
    "    sheet = xldoc.add_sheet(\"Sheet1\", cell_overwrite_ok=True)\n",
    "    # Iterating and saving the data to sheet\n",
    "    for i, row in enumerate(data):\n",
    "        # Two things are done here\n",
    "        # Removeing the '\\n' which comes while reading the file using io.open\n",
    "        # Getting the values after splitting using '\\t'\n",
    "        for j, val in enumerate(row.replace('\\n', '').split('\\t')):\n",
    "            sheet.write(i, j, val)\n",
    "    \n",
    "    xldoc.save(filename)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "def netmhcpan_summarize(protein, status, mers, hla_list):\n",
    "\n",
    "    # Path to netMHCpan results\n",
    "    results_path = \"../netmhcpan_binding_result/{}\".format(protein)\n",
    "\n",
    "    all_results = []\n",
    "\n",
    "    # For given k-mer, retrieve all the netMHCpan results and summarize\n",
    "    for hla in hla_list:\n",
    "\n",
    "        hla_results = []\n",
    "\n",
    "        for i in range(len(mers)):\n",
    "            hla_mer_result = \"{}/{}_{}_{}_mers_{}.xls\".format(results_path, protein, status, mers[i], hla.replace(\":\",\"_\"))\n",
    "\n",
    "            # xls file outputted from netMHCpan is somehow corrupted...\n",
    "            # try to recover it.\n",
    "            try:\n",
    "                recover_xls(hla_mer_result)\n",
    "            except:\n",
    "                pass\n",
    "\n",
    "            hla_mer_result = pd.ExcelFile(hla_mer_result).parse('Sheet1')\n",
    "\n",
    "            col_names = hla_mer_result.iloc[0, :]\n",
    "            hla_mer_result = hla_mer_result.iloc[1:, :]\n",
    "            hla_mer_result.columns = col_names\n",
    "            hla_mer_result.pop(\"Ave\")\n",
    "\n",
    "            hla_results.append(hla_mer_result)\n",
    "        \n",
    "        hla_results = pd.concat(hla_results)\n",
    "        \n",
    "        # NB = 0, no binding, NB =1, there is a binding (either weak or strong, depending on EL_Rank)\n",
    "        hla_results[\"NB\"] = [hla_results[\"NB\"].iloc[i].strip() for i in range(len(hla_results))]\n",
    "\n",
    "        binding_results = hla_results.loc[hla_results[\"NB\"] == \"1\"]\n",
    "\n",
    "        #HLA-A:02:01 -> HLA-A*02:01 put a star in between, for matching formats for population coverage analysis..\n",
    "        hla_star = hla[:5] + \"*\" + hla[5:]\n",
    "        hla_column = [hla_star] * len(binding_results)\n",
    "        binding_results.insert(5, \"HLA\", hla_column)\n",
    "        \n",
    "        binding_strength = []\n",
    "        for value in binding_results[\"EL_Rank\"]:         \n",
    "            if float(value) <= 0.5:\n",
    "                binding_strength.append(\"SB\")\n",
    "            else:\n",
    "                binding_strength.append(\"WB\")\n",
    "        \n",
    "        binding_results.insert(6, \"Binding\", binding_strength)\n",
    "        binding_results.pop(\"NB\")\n",
    "\n",
    "        all_results.append(binding_results)\n",
    "\n",
    "    all_results = pd.concat(all_results).sort_values('Peptide')\n",
    "\n",
    "    # Worst case. No binding cases at all. . . SAD\n",
    "    if all_results.empty:\n",
    "        return all_results\n",
    "\n",
    "    return all_results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [],
   "source": [
    "def immunogenicity_result(protein, status):\n",
    "    \"\"\"\n",
    "    Input\n",
    "        - netmhc_summary = pd table of netMHCpan summary\n",
    "    \"\"\"\n",
    "    netmhc_summary = pd.read_csv(\"../MHC_neoIM_summary/{}_{}_epitopes_summary.csv\".format(protein, status))\n",
    "    unique_peptides = list(set(netmhc_summary['Peptide'].tolist()))\n",
    "\n",
    "    # Immunogenicity result for corresponding proteins\n",
    "    neoIM_file = '../Ju_results2/{}_neoIM.csv'.format(protein)\n",
    "    neoIM = pd.read_csv(neoIM_file)\n",
    "\n",
    "    neoim_results = []\n",
    "    IDs = []\n",
    "\n",
    "    for pep in unique_peptides:\n",
    "\n",
    "        neoim_score = neoIM.loc[neoIM['epitope'] == pep]['neoIM_score']\n",
    "\n",
    "        if len(neoim_score) == 0:\n",
    "            neoim_results.append(\"-\")\n",
    "        else:\n",
    "            neoim_results.append(neoim_score.iloc[0])\n",
    "        \n",
    "        identity = netmhc_summary.loc[netmhc_summary['Peptide'] == pep]['ID'].tolist()[0]\n",
    "        IDs.append(identity)\n",
    "    \n",
    "    data = {\n",
    "        \"epitope\": unique_peptides,\n",
    "        \"ID\": IDs,\n",
    "        \"neoIM_score\": neoim_results\n",
    "    }\n",
    "\n",
    "    immuno_pd = pd.DataFrame(data=data).sort_values(\"neoIM_score\", ascending=False)\n",
    "\n",
    "    return immuno_pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>epitope</th>\n",
       "      <th>ID</th>\n",
       "      <th>neoIM_score</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>FNMLSTVLGV</td>\n",
       "      <td>cand_82_o</td>\n",
       "      <td>0.98385</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>GMFNMLSTVL</td>\n",
       "      <td>cand_99_o</td>\n",
       "      <td>0.98105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>62</th>\n",
       "      <td>NMLSTVLGV</td>\n",
       "      <td>cand_104_o</td>\n",
       "      <td>0.97295</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>GMFNMLSTV</td>\n",
       "      <td>cand_107_o</td>\n",
       "      <td>0.97220</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>101</th>\n",
       "      <td>FVANFSMEL</td>\n",
       "      <td>cand_307_o</td>\n",
       "      <td>0.96910</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>NTMTKDAER</td>\n",
       "      <td>cand_8</td>\n",
       "      <td>0.32945</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>RNRSILNTS</td>\n",
       "      <td>cand_333</td>\n",
       "      <td>0.32895</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>TKDAERGKL</td>\n",
       "      <td>cand_11</td>\n",
       "      <td>0.32195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>DAERGKLKR</td>\n",
       "      <td>cand_46</td>\n",
       "      <td>0.31680</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>QLFIKDYRY</td>\n",
       "      <td>cand_226</td>\n",
       "      <td>0.03215</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>102 rows × 3 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "        epitope          ID  neoIM_score\n",
       "0    FNMLSTVLGV   cand_82_o      0.98385\n",
       "37   GMFNMLSTVL   cand_99_o      0.98105\n",
       "62    NMLSTVLGV  cand_104_o      0.97295\n",
       "24    GMFNMLSTV  cand_107_o      0.97220\n",
       "101   FVANFSMEL  cand_307_o      0.96910\n",
       "..          ...         ...          ...\n",
       "32    NTMTKDAER      cand_8      0.32945\n",
       "21    RNRSILNTS    cand_333      0.32895\n",
       "8     TKDAERGKL     cand_11      0.32195\n",
       "9     DAERGKLKR     cand_46      0.31680\n",
       "27    QLFIKDYRY    cand_226      0.03215\n",
       "\n",
       "[102 rows x 3 columns]"
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "immunogenicity_result(\"PB1\", 'exp')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [],
   "source": [
    "proteins = [\"HA\", \"NA\", \"M1\", \"M2\", \"PB1\", \"PB2\", \"PA\", \"NP\", \"NS1\", \"NS2\"]\n",
    "\n",
    "for prot in proteins:\n",
    "    try:\n",
    "        immuno_cons = immunogenicity_result(prot, 'cons')\n",
    "        immuno_cons.to_csv(\"../MHC_neoIM_summary/neoIM_summary/{}_neoim_cons.csv\".format(prot), index=False)\n",
    "    except:\n",
    "        pass\n",
    "    try:\n",
    "        immuno_exp = immunogenicity_result(prot, 'exp')\n",
    "        immuno_exp.to_csv(\"../MHC_neoIM_summary/neoIM_summary/{}_neoim_exp.csv\".format(prot), index=False)\n",
    "    except:\n",
    "        pass"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [],
   "source": [
    "def output_for_population_coverage(outfile_path, summary):\n",
    "\n",
    "    unique_peptides = sorted(set(summary[\"Peptide\"].tolist()))\n",
    "\n",
    "    for peptide in unique_peptides:\n",
    "        hlas = summary.loc[summary['Peptide'] == peptide]['HLA']\n",
    "        hlas = \", \".join(hlas.tolist())\n",
    "\n",
    "        outfile_pep = \"{}/{}.txt\".format(outfile_path, peptide)\n",
    "\n",
    "        with open(outfile_pep, 'w') as output:\n",
    "            output.write(peptide + \", \" + hlas)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [],
   "source": [
    "def output_for_popcoverage(protein, status):\n",
    "    netmhc_summary = pd.read_csv(\"../MHC_neoIM_summary/{}_{}_epitopes_summary.csv\".format(protein, status))\n",
    "    immuno_summary = pd.read_csv(\"../MHC_neoIM_summary/neoIM_summary/{}_neoim_{}.csv\".format(protein, status))\n",
    "\n",
    "    epitopes = immuno_summary['epitope'].tolist()\n",
    "\n",
    "    for pep in epitopes:\n",
    "        immunogenicity = list(set(immuno_summary.loc[immuno_summary['epitope'] == pep]['neoIM_score'].tolist()))[0]\n",
    "        \n",
    "        if immunogenicity >= 0.64:\n",
    "            hlas = netmhc_summary.loc[netmhc_summary['Peptide'] == pep]['HLA']\n",
    "            hlas = \", \".join(hlas.tolist())\n",
    "\n",
    "            outfile_path = \"../MHC_neoIM_summary/immunogenic_epitopes/\" + protein\n",
    "\n",
    "            if not os.path.exists(outfile_path):\n",
    "                os.mkdir(outfile_path)\n",
    "            \n",
    "            status_path = \"{}/{}\".format(outfile_path, status)\n",
    "\n",
    "            if not os.path.exists(status_path):\n",
    "                os.mkdir(status_path)\n",
    "\n",
    "            filename = \"{}/{}.txt\".format(status_path, pep)\n",
    "            with open(filename, 'w') as output:\n",
    "                output.write(pep + '\\t' + hlas)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [],
   "source": [
    "for prot in proteins:\n",
    "    try:\n",
    "        output_for_popcoverage(prot, 'cons')\n",
    "    except:\n",
    "        pass\n",
    "    \n",
    "    try:\n",
    "        output_for_popcoverage(prot, 'exp')\n",
    "    except:\n",
    "        pass"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "HA\n",
      "NA\n",
      "M1\n",
      "M2\n",
      "PB1\n",
      "PB2\n",
      "PA\n",
      "NP\n",
      "NS1\n"
     ]
    }
   ],
   "source": [
    "prots = ['HA', 'NA', 'M1', 'M2', \"PB1\", \"PB2\", \"PA\", \"NP\", \"NS1\"]\n",
    "\n",
    "import pandas as pd\n",
    "\n",
    "for prot in prots:\n",
    "    print(prot)\n",
    "\n",
    "    # Defining path for outputs\n",
    "    outfile_prot = \"../mhcbinding_text/\" + prot\n",
    "\n",
    "    if not os.path.exists(outfile_prot):\n",
    "        os.mkdir(outfile_prot)\n",
    "\n",
    "    outfile_cons = \"{}/conserved\".format(outfile_prot)\n",
    "    outfile_exp = \"{}/experimental\".format(outfile_prot)\n",
    "\n",
    "    if not os.path.exists(outfile_cons):\n",
    "        os.mkdir(outfile_cons)\n",
    "    if not os.path.exists(outfile_exp):\n",
    "        os.mkdir(outfile_exp)\n",
    "\n",
    "    conserved_epitopes, known_mers, exp_epitopes, exp_mers = fetch_files(prot)\n",
    "\n",
    "    if len(known_mers) > 0:\n",
    "        cons_summary = netmhcpan_summarize(prot, 'conserved', known_mers, HLA_list)\n",
    "        if not cons_summary.empty:\n",
    "            output_for_population_coverage(outfile_cons, cons_summary)\n",
    "            cons_summary.to_csv(\"../MHC_neoIM_summary/\" + prot + \"_cons_epitopes_summary.csv\", index=False)\n",
    "\n",
    "    if len(exp_mers) > 0:\n",
    "        exp_summary = netmhcpan_summarize(prot, 'experimental', exp_mers, HLA_list)\n",
    "        if not exp_summary.empty:\n",
    "            output_for_population_coverage(outfile_exp, exp_summary)\n",
    "            exp_summary.to_csv(\"../MHC_neoIM_summary/\" + prot + \"_exp_epitopes_summary.csv\", index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "* No result found! *\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "0"
      ]
     },
     "execution_count": 35,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "iedb_population = '../population_coverage/calculate_population_coverage.py'\n",
    "files = \"../MHC_neoIM_summary/immunogenic_epitopes/HA/exp/GMVDGWYGF.txt\"\n",
    "\n",
    "os.system('python {} -p Japan -c I -f {}'.format(iedb_population, files))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "class I\n",
      "population/area\tcoverage\taverage_hit\tpc90\n",
      "World\t93.16%\t7.01\t2.13\n",
      "average\t93.16%\t7.01\t2.13\n",
      "standard_deviation\t0.0%\t0.0\t0.0\n",
      "\n",
      "population/area\tepitope_hits\tpercent_individuals\tcumulative_coverage\n",
      "World\t0\t6.84\t100.0\n",
      "World\t1\t2.17\t93.16\n",
      "World\t2\t7.46\t90.99\n",
      "World\t3\t3.9\t83.53\n",
      "World\t4\t6.71\t79.63\n",
      "World\t5\t13.14\t72.92\n",
      "World\t6\t7.95\t59.79\n",
      "World\t7\t11.42\t51.84\n",
      "World\t8\t5.84\t40.41\n",
      "World\t9\t7.73\t34.57\n",
      "World\t10\t6.6\t26.85\n",
      "World\t11\t5.49\t20.25\n",
      "World\t12\t4.4\t14.76\n",
      "World\t13\t2.83\t10.36\n",
      "World\t14\t2.54\t7.53\n",
      "World\t15\t1.46\t4.99\n",
      "World\t16\t1.3\t3.52\n",
      "World\t17\t0.82\t2.22\n",
      "World\t18\t0.57\t1.4\n",
      "World\t19\t0.31\t0.83\n",
      "World\t20\t0.22\t0.52\n",
      "World\t21\t0.13\t0.3\n",
      "World\t22\t0.08\t0.17\n",
      "World\t23\t0.05\t0.09\n",
      "World\t24\t0.02\t0.04\n",
      "World\t25\t0.01\t0.02\n",
      "World\t26\t0.0\t0.01\n",
      "World\t27\t0.0\t0.01\n",
      "World\t28\t0.0\t0.0\n",
      "World\t29\t0.0\t0.0\n",
      "World\t30\t0.0\t0.0\n",
      "World\t31\t0.0\t0.0\n",
      "\n",
      "\n",
      "* A plot has been generated in '../' directory with <population>_<mhc_class> suffixed.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "0"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "population_coverage = '../population_coverage/calculate_population_coverage.py'\n",
    "\n",
    "prot = 'HA'\n",
    "results_path = \"../netmhcpan_tool_result/{}\".format(prot)\n",
    "input_pop = \"{}/../{}_population_input.txt\".format(results_path, prot)\n",
    "\n",
    "pop_analysis = \"python {} -p World -c I -f {} --plot ../\".format(population_coverage, input_pop)\n",
    "os.system(pop_analysis)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>epitopes</th>\n",
       "      <th>HLA_alleles</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>ALQLLLEV</td>\n",
       "      <td>HLA-A*02:01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>EEIRWLIEEV</td>\n",
       "      <td>HLA-B*44:03</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>GMITQFESL</td>\n",
       "      <td>HLA-A*02:01,HLA-A*32:01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>GMITQFESLK</td>\n",
       "      <td>HLA-A*03:01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>IRWLIEEV</td>\n",
       "      <td>HLA-C*06:02</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>KFEEIRWL</td>\n",
       "      <td>HLA-C*04:01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>KFEEIRWLI</td>\n",
       "      <td>HLA-A*32:01,HLA-C*04:01,HLA-A*23:01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>LLLEVEQEI</td>\n",
       "      <td>HLA-C*04:01,HLA-A*32:01,HLA-C*05:01,HLA-A*02:01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>LQLLLEVEQEI</td>\n",
       "      <td>HLA-A*02:01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>MITQFESLK</td>\n",
       "      <td>HLA-A*03:01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>QALQLLLEV</td>\n",
       "      <td>HLA-B*51:01,HLA-C*16:01,HLA-C*15:02,HLA-A*02:0...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>QKFEEIRWL</td>\n",
       "      <td>HLA-C*07:02,HLA-C*06:02,HLA-B*38:01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>QLLLEVEQEI</td>\n",
       "      <td>HLA-A*02:01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>SLKLYRDSL</td>\n",
       "      <td>HLA-B*07:02</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>TFMQALQL</td>\n",
       "      <td>HLA-C*04:01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>TFMQALQLL</td>\n",
       "      <td>HLA-C*06:02,HLA-B*38:01,HLA-A*23:01,HLA-C*07:0...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>TQFESLKL</td>\n",
       "      <td>HLA-B*38:01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>TQFESLKLY</td>\n",
       "      <td>HLA-C*07:02,HLA-C*12:03,HLA-A*01:01,HLA-B*44:0...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>TQFESLKLYR</td>\n",
       "      <td>HLA-A*03:01</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "       epitopes                                        HLA_alleles\n",
       "0      ALQLLLEV                                        HLA-A*02:01\n",
       "1    EEIRWLIEEV                                        HLA-B*44:03\n",
       "2     GMITQFESL                            HLA-A*02:01,HLA-A*32:01\n",
       "3    GMITQFESLK                                        HLA-A*03:01\n",
       "4      IRWLIEEV                                        HLA-C*06:02\n",
       "5      KFEEIRWL                                        HLA-C*04:01\n",
       "6     KFEEIRWLI                HLA-A*32:01,HLA-C*04:01,HLA-A*23:01\n",
       "7     LLLEVEQEI    HLA-C*04:01,HLA-A*32:01,HLA-C*05:01,HLA-A*02:01\n",
       "8   LQLLLEVEQEI                                        HLA-A*02:01\n",
       "9     MITQFESLK                                        HLA-A*03:01\n",
       "10    QALQLLLEV  HLA-B*51:01,HLA-C*16:01,HLA-C*15:02,HLA-A*02:0...\n",
       "11    QKFEEIRWL                HLA-C*07:02,HLA-C*06:02,HLA-B*38:01\n",
       "12   QLLLEVEQEI                                        HLA-A*02:01\n",
       "13    SLKLYRDSL                                        HLA-B*07:02\n",
       "14     TFMQALQL                                        HLA-C*04:01\n",
       "15    TFMQALQLL  HLA-C*06:02,HLA-B*38:01,HLA-A*23:01,HLA-C*07:0...\n",
       "16     TQFESLKL                                        HLA-B*38:01\n",
       "17    TQFESLKLY  HLA-C*07:02,HLA-C*12:03,HLA-A*01:01,HLA-B*44:0...\n",
       "18   TQFESLKLYR                                        HLA-A*03:01"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "population_coverage_input"
   ]
  }
 ],
 "metadata": {
  "interpreter": {
   "hash": "1bd06f5ea03ec233ea613593a8abdc9f0315a389ccaf2775c94be72fffa4b7be"
  },
  "kernelspec": {
   "display_name": "Python 3.8.12 64-bit ('base': conda)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.12"
  },
  "orig_nbformat": 4
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
